resources/mentors.py

```diff
@@ -184,15 +185,16 @@ def post(self):
                 conn.close()
 
 class GetMentorQuestions(Resource):
+    @jwt_required
 
     # @jwt_required
-    def post(self):
```

Comment:
this needs to be a post and NOT a get. This will ruin Unity's ability to interact with the server